frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -157,7 +193,7 @@ export function MainInfoSection(props: MainInfoSectionProps) {
         <SectionHeader
           title={title || (resource ? resource.kind : '')}
           headerStyle={headerStyle}
-          actions={allActions}
+          actions={allActions.filter(action => !!action)}
```

Comment:
Should this be done inside SectionHeader? I'm wondering, because this seems to be an API change one what SectionHeader.actions accepts. If it is an API change in this way, it could be fixed by having SectionHeader take care of this filtering.

Reply:
I don't think this is an API change. Actually things should work without this change. I was trying to avoid passing [null, null, null] for example but I don't think this does that even, because only when rendering do the components turn to null.
Due to this we do get empty <div> in the resulting HTML but I don't think we can avoid it...